src/Command/InstallProjectCommand.php

```diff
@@ -0,0 +1,213 @@
+<?php
+namespace App\Command;
+
+use Exception;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Process\Process;
+
+/**
+ * Class InstallProjectCommand
+ *
+ * @package App\Command
+ * @desc By running this command you are installing the Take Home Task project
+ * @desc
+ */
+class InstallProjectCommand extends Command
+{
+    protected static $defaultName = 'gimmemore:install';
+
+    private SymfonyStyle $io;
+
+    private string $projectDir;
+
+    private int $timeout = 600;
+
+    public function __construct(string $projectDir)
+    {
+        set_time_limit(0);
+
+        parent::__construct();
+        $this->projectDir = $projectDir;
+    }
+
+    protected function configure()
+    {
+        $this->setDescription('Install gimmemore project');
+    }
+
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int
+     * @throws Exception
+     */
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $input->setInteractive(false);
+
+        $this->io = $io;
+
+        try {
+            $this->runComposerInstall();
+            $this->runDropDatabase();
+            $this->runCreateDatabase();
+            $this->runDoctrineSchemaUpdate();
+            $this->runDoctrineFixturesLoad();
+            $this->runYarnInstall();
+            $this->runYarnBuild();
+
+            return Command::SUCCESS;
+        } catch (Exception $e) {
+            throw $e;
+        }
+    }
+
+    /**
+     * @return void
+     */
+    protected function runComposerInstall(): void
+    {
+        $process = new Process(['composer', 'install'], $this->projectDir);
+
+        $this->runProcessBlockCommands($process, 'Trying to execute composer install ...');
+    }
+
+
+    /**
+     * @return void
+     */
+    protected function runDropDatabase(): void
+    {
+        $process = new Process(['bin/console', 'doctrine:database:drop', '--if-exists', '--force'], $this->projectDir);
+
+        $this->runProcessBlockCommands($process, 'Trying to drop database ...');
+    }
+
+    /**
+     * @return void
+     */
+    protected function runCreateDatabase(): void
+    {
+        $process = new Process(['bin/console', 'doctrine:database:create'], $this->projectDir);
+
+        $this->runProcessBlockCommands($process, 'Trying to create database ...');
+    }
+
+    /**
+     * @return void
+     */
+    protected function runDoctrineSchemaUpdate(): void
+    {
+        $process = new Process(
+            [
+                'bin/console',
+                'doctrine:schema:update',
+                '--no-interaction',
+                '--force',
```

Comment:
I can see a heavy legacy of the ORO platform here. 
Given initial developers came from the Magento background, it's natural they would have the "install" command. 
Neat idea, but it rarely works in practice.

In any case, we should not have `--force` for prod, right? :)

Reply:
Yeap, you're right. I loved the way they decided to help developers running the project with a fresh database ... . That's why I did this to help you install the project much more easier .